### DISCUSSION TITLE
[Feature] Estimates the location of pictures taken without GPS information

### DISCUSSION THREAD

**Discussion by cttlee** [immich-app/immich#5626] [Feature Request]
### The feature

Estimates the location of pictures taken without GPS information by extrapolating it from the location of other pictures taken on the same day.

Many digital cameras do not have GPS, it's useful for pictures taken from both phone and camera at same day.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1675
